src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/Safeguarding.cshtml.cs

```diff
@@ -14,4 +14,13 @@ public void OnGet(string id, string name)
         Id = id;
         Name = name;
     }
+
+    public IActionResult OnPost(string id, string name) 
```

Comment:
It looks like the source form in the safeguarding page PR doesn't have the service id and name?

If we passed the service id and name in the URL, we could cut out a POST and redirect and just send them straight to the consent form.

Reply:
Uses anchor now which appears as a button